pfb-analysis/connectivity/census_blocks.sql

```diff
@@ -4,17 +4,117 @@
 -- code to be run on table that has
 -- been imported directly from US Census
 -- blkpophu file
+-- :nb_output_srid psql var must be set before running this script,
+--      e.g. psql -v nb_output_srid=2249 -f census_blocks.sql
 ----------------------------------------
 
-ALTER TABLE neighborhood_census_blocks ADD COLUMN IF NOT EXISTS pop_low_stress INT;
-ALTER TABLE neighborhood_census_blocks ADD COLUMN IF NOT EXISTS pop_high_stress INT;
-ALTER TABLE neighborhood_census_blocks ADD COLUMN IF NOT EXISTS emp_low_stress INT;
-ALTER TABLE neighborhood_census_blocks ADD COLUMN IF NOT EXISTS emp_high_stress INT;
-ALTER TABLE neighborhood_census_blocks ADD COLUMN IF NOT EXISTS schools_low_stress INT;
-ALTER TABLE neighborhood_census_blocks ADD COLUMN IF NOT EXISTS schools_high_stress INT;
-ALTER TABLE neighborhood_census_blocks ADD COLUMN IF NOT EXISTS rec_low_stress INT;
-ALTER TABLE neighborhood_census_blocks ADD COLUMN IF NOT EXISTS rec_high_stress INT;
+ALTER TABLE neighborhood_census_blocks DROP COLUMN IF EXISTS road_ids;
```

Comment:
We don't currently export this table. Is this information we'll need to export or will this information be aggregated and encapsulated in some way to the `neighborhood_overall_scores` table?